src/main/java/com/batix/rundeck/plugins/AnsibleResourceModelSource.java

```diff
@@ -290,12 +294,12 @@ public INodeSet getNodes() throws ResourceModelSourceException {
           // ansible_system     = Linux   = osFamily in Rundeck
           // ansible_os_family  = Debian  = osName in Rundeck
 
-          if (root.has("ansible_system")) {
-            node.setOsFamily(root.get("ansible_system").getAsString());
+          if (root.has("ansible_os_family")) {

```

Comment:
Are you sure this mapping for OsFamily and OsName to Ansible facts is correct (haven't tested it)?